test/integration/setting_js_test.rb

```diff
@@ -29,4 +29,78 @@ class SettingJSTest < IntegrationTestWithJavascript
     assert_index_page(settings_path, "Settings", false, true, false)
     assert page.has_link?("My Pretty Setting Label", :href => "#category_label_test_settings_tab")
   end
+
+  test "general headers values" do
+    visit settings_path
+
+    expect page.has_content?('Name')
+    expect page.has_content?('Value')
+    expect page.has_content?('Description')
```

Comment:
https://guides.rubyonrails.org/testing.html#capybara-assertions says there is `assert_text(type, text)`. I think that would be a better matcher, with the downside that you need to specify which type the element has.

Though when I look at https://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Matchers it's simply `assert_text(text)`. https://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Matchers#assert_text-instance_method clarifies that you can use both.

So you can use
```suggestion
    assert_text 'Name'
    assert_text 'Value'
    assert_text 'Description'
```

I'll also note that `expect` is more commonly used with RSpec but in Foreman we use minitest (for better or worse).